src/app/modules/shared/utils/project-storage.util.ts

```diff
@@ -0,0 +1,30 @@
+import { Project } from '../models/project.model';
+import { PROJECTS_KEY_FOR_LOCAL_STORAGE } from '../../../../environments/environment';
+
+const projectsKey = PROJECTS_KEY_FOR_LOCAL_STORAGE;
+
+export function updateProjectStorage(serverProjects: Project[]): void {
+  const storageProjects: Project[] = getProjectsOnStorage(projectsKey);
+  const isServerProjectsNotEmpty = serverProjects && serverProjects.length !== 0;
```

Comment:
you could use lodash library and use isEmpty utility to validate both cases, you can review about this library here https://lodash.com and the implementation here https://www.npmjs.com/package/lodash

Reply:
Good suggestion Gustavo, it was implemented only because lodash is already used in the project, however, for such a simple verification that can be performed with the base language, I consider unnecessary the implementation of the library, besides adding a dependency to it in case you want to replace it in the future.